### DISCUSSION TITLE
【求助】怎么换头像啊？

### DISCUSSION THREAD

**Discussion by NOVAQUQ** [share121/inter-knot#209] [其他]
![1701149023910](https://github.com/user-attachments/assets/88ba63e6-c402-4dae-9356-429c5dff2f22)
怎么换头像啊？我到底怎么换头像啊？

**Comment by JingLu170** [comment]
你点击你的头像，进主页就可以换了

**Reply by NOVAQUQ** [reply]
肥肠感谢

**Comment by gjbctuj** [comment]
![Screenshot_2024-08-09-21-58-23-46_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/672fe48e-5018-4400-8411-034df7da7e01)

**Reply by NOVAQUQ** [reply]
太感谢了，好人一生平安Orz

**Comment by yuanzidamn** [comment]
感谢，有点贴吧味了

**Comment by NOVAQUQ** [comment]
换好辣！( *ˊᵕˋ)✩︎‧₊
